KmsClient.js

```diff
@@ -57,11 +54,8 @@ export class KmsClient {
     _assert(type, 'type', 'string');
     _assert(invocationSigner, 'invocationSigner', 'object');
 
-    const keySuite = cryptoLd.suites.get(type);
-    const contextUrl = keySuite.CONTEXT_URL;
-
     const operation = {
-      '@context': contextUrl,
+      '@context': CONTEXT_URL,
```

Comment:
This won't be sufficient because it doesn't have the context for the `type`. We can't delete the above code that includes that -- we need both the webkms context and the suite-specific context (in that order in an array).

Reply:
Also, if tests are passing with this, we should add a test to ensure that they don't.

Reply:
Fixed; now including suite-specific and KMS context both. (Also for symmetric key types.)

Reply:
Oh wait, no, I'm thinking of the other repo. Fixing this one now.

Reply:
addressed here 23de7e3